src/common/hooks/useDateTime.ts

```diff
@@ -37,16 +31,18 @@ export function useDateTime() {
       timeFormat || companyTimeFormat
     }`;
 
+    if (formatOnlyTime && typeof date === 'number') {
+      return dayjs.unix(date).format(timeFormat || companyTimeFormat);
+    }
+
+    if (formatOnlyTime && typeof date !== 'number') {
+      return dayjs(date).format(timeFormat || companyTimeFormat);
+    }
+
     if (typeof date === 'number') {
-      return dayjs
-        .utc(date)
-        .tz(timeZone || companyTimeZone)
-        .format(finalFormat);
+      return dayjs.unix(date).format(finalFormat);
     }
 
-    return dayjs
-      .utc(date)
-      .tz(timeZone || companyTimeZone)
```

Comment:
@beganovich After our quick discussion in the group about time zones, I'm now aware that we don't need to add timezone here. So, I've removed it.

Reply:
But this hook is used in different places? Is the behavior same after your change?

Reply:
@beganovich Currently, the hook is only used in three places for formatting the `next_send_datetime` column value in Recurring Invoices. While we have many similar column formatting instances throughout the tables, this particular one formats both date and time.

The behavior will change as it will no longer use the company timezone. However, this aligns with our existing formatting logic, as we don't use company timezones for any other date or time column formatting in the application. The `dayjs` will take the timezone from the browser and format it into that timezone, which is how we handle all other formatting.

Basically change aligns logic with the rest of the app.